docs/rules/prefer-event-key.md

```diff
@@ -0,0 +1,37 @@
+# Enforce the use of `event.key` instead of `event.keyCode`
+
+Enforces the use of [`KeyboardEvent#key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) over [`KeyboardEvent#keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) which is deprecated. The `key` property is also more semantic and readable.
+
+
```

Comment:
Please describe that it's fixable and what is fixable. See the other docs for how to write it.